### PR TITLE
Bump to version 3.5.0

### PR DESCRIPTION
**Pull request by 1445601153**
Bump to version 3.5.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -95,6 +113,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 
 - Implement fs services.
 
+[v3.5.0]: https://github.com/beyondstorage/go-service-fs/compare/v3.3.0...v3.5.0
```
Comment (abyss-w) [inline CHANGELOG.md:116]: `v3.3.0` should be changed to `v3.4.0`.

Comment (1445601153) [reply CHANGELOG.md]: It was really careless of me.

**Review by abyss-w — APPROVED** [review]
Most LGTM

**Comment by JinnyYi** [comment]
Maybe we can checkout a new branch so there are no other commit messages.

**Pull request merged**